.github/actions/apt-x64/action.yml

```
@@ -0,0 +1,67 @@
+name: apt
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+        sudo apt-get install \
```

```suggestion
        set -x
        sudo apt-get update
        sudo apt-get install \
```
You might've missed by suggestion for `apt-get update` in the previous PR.

---

Ah right, thanks